doc/configuration/introduction.md

```diff
@@ -32,7 +32,7 @@ bootstrap_from_trusted_peers: false
 
 p2p:
   public_address: "/ip4/X.X.X.X/tcp/Y" # This should match your public IP address (X) and port number (Y)
-  #listen_address: /ip4/0.0.0.0/tcp/Z
+  #listen: 0.0.0.0:Y
```

Comment:
Any particular reason for that change? I mean both moving away from multiaddr and the name change.

Reply:
For a listen address a multiaddr does not make a lot of sense. As for the name, it is just to uniform with the rest config, where the field is also called just `listen`